vue/components/ui/selector-platforme/selector-platforme.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div class="selector-container">
+        <svg
+            class="select_arrow"
+            fill="#7b7b7b"
+            height="24"
+            viewBox="0 0 24 24"
+            width="20"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z" />
+            <path d="M0-.75h24v24H0z" fill="none" />
+        </svg>
+        <select class="selector" required>
+            <option selected disabled value="">
+                Select one
+            </option>
+            <option
+                class="option"
+                v-bind:value="item.value"
+                v-for="item in items.filter(v => v !== null && v !== undefined)"
+                v-bind:id="item.id"
+                v-bind:key="item.id"
+            >
+                {{ item.text }}
+            </option>
+        </select>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.selector-container {
+    display: inline-block;
+    position: relative;
+}
+
+.select_arrow {
+    pointer-events: none;
+    position: absolute;
+    right: 10px;
+    top: 7px;
+}
+
+.selector {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    background: #f2f2f2 none repeat scroll 0% 0% / auto padding-box border-box;

```

Comment:
I don't have the slightest clue what "/ auto padding-box border-box" is but probably you can delete it :p